docs/advanced/nat.md

```diff
@@ -1,14 +1,59 @@
+There might be some cases, especially when working in lab environments, where **it is not possible to configure the gateway server service as `LoadBalancer`**, requiring it to be exposed as a `NodePort` service instead.
+However, the nodes of the cluster might not be directly reachable, as for example, they are behind a NAT or a phisycal load balancer.
 
-Under the `networking.gatewayTemplates.server.service.annotations` key, you can set the following annotations:
+In this documentation page, we are going to describe how to configure Liqo in these scenarios.
 
-* **liqo.io/override-address**: the public IP address of the NAT firewall.
-* **liqo.io/override-port**: the public port of the NAT firewall.
+## Option A: Reverse Liqo network
 
-```{admonition} Tip
-In case you need to have multiple gateways behind the same NAT firewall, you need to override the port for each peer using the `--server-port` flag at peering time.
-This flag works with `liqoctl peer` and `liqoctl network connect` commands.
+![The provider is behind a NAT](../_static/images/advanced/nat/provider-nat.svg)
+
+By default using the `peer` command, the gateway server is configured on the provider cluster.
+However, there may be cases where the provider cluster's nodes are not directly reachable, such as when they are behind a NAT, while the consumer cluster is directly accessible.
+For instance, in the image above, cluster 2 is behind a NAT and is therefore not directly reachable.
+
+To fix this issue, we can swap the roles of the gateways, configuring the client on the cluster provider and the server on the consumer.
+To do so, you need to use [manual peering](./manual-peering.md), setting the inter-cluster network up separately.
+
+![The gateway server has been on the consumer side](../_static/images/advanced/nat/consumer-nat.svg)
+
+In this case, once the gateway server has been moved to cluster 1, we are able to configure the Liqo networking without the need to configure port-mapping on the NAT, as the gateway server is directly reachable.
+
+In the case you would like to keep the gateway server on the provider, you can follow the instructions on the next paragraph.
+
+## Option B: configure the gateway server to be reachable through the NAT (or physical load balancer)
+
+The solution presented in this section allows to reach the gateway server even when it is behind a NAT (or, for example, a physical load balancer) which prevents it from being directly reachable.
+
+![The provider is behind a NAT](../_static/images/advanced/nat/port-address-override.svg)
+
+In this case, **you must configure port-mapping on the NAT** so that the traffic directed on a specific port of the NAT can be forwarded to one of nodes of the cluster.
+
+**When network is configured via `liqoctl` commands**, if the service is of type `NodePort`, Liqo automatically uses the external IPs of the nodes to configure the gateway client.
```

Comment:
I do not understand the term "automatically". If it is automatic, it means that the user has to do nothing to use the "external IP". However, from the next sentence, it seems that the user has to do something to let liqoctl knowing that we're in that situation. This means that it not automatic.
Am I right?
If so, I suggest to rearrange a bit the sentence to make things clearer.